### PR TITLE
Update release-notes.md

### PR DESCRIPTION
**Pull request by christopherasnider**
- added Web (blip) 1.12.2, 1.12.3
- added Uploader 2.7.6

### CODE REVIEW

**Comment by christopherasnider** [comment]
@gniezen My 🎁 to you

File: _docs/release-notes.md [hunk 1]
```diff
@@ -24,6 +24,21 @@ Visit [https://app.tidepool.org](https://app.tidepol.org) to see the results of
```
Comment (gniezen) [inline _docs/release-notes.md]: I just noticed there's a type in a URL in this doc (not part of the changes): app.tidepol.org

File: _docs/release-notes.md [hunk 2]
```diff
@@ -24,6 +24,21 @@ Visit [https://app.tidepool.org](https://app.tidepol.org) to see the results of
...
 Don't have a Tidepool account? Visit [tidepool.org/signup](https://tidepool.org/signup) to create your free Tidepool account.  
 
+### 1.12.3 (Released 2018-09-12)
+We added timestamps to CGM data in your Daily View. With so many dots to keep track of, this should help keep things organized.  
+[Feature Requirements](https://trello.com/c/QWz76ymx/16-s-data-viz-daily-view-cgm-add-timestamp-in-hover)
+
+[Code Update](https://github.com/tidepool-org/blip/releases/tag/v1.12.3)
+
+### 1.12.2 (Released 2018-09-11)
+Our new Content Security Policy caused issues for people trying to access Tidepool on Safari. Sorry about that one, that's been fixed.
```
Comment (gniezen) [inline _docs/release-notes.md]: Should we be more specific here to indicate that it's to create an account, as you can't actually "access" Tidepool on Safari?

File: _docs/release-notes.md [hunk 3]
```diff
@@ -24,6 +24,21 @@ Visit [https://app.tidepool.org](https://app.tidepol.org) to see the results of
...
+We added timestamps to CGM data in your Daily View. With so many dots to keep track of, this should help keep things organized.  
+[Feature Requirements](https://trello.com/c/QWz76ymx/16-s-data-viz-daily-view-cgm-add-timestamp-in-hover)
+
+[Code Update](https://github.com/tidepool-org/blip/releases/tag/v1.12.3)
+
+### 1.12.2 (Released 2018-09-11)
+Our new Content Security Policy caused issues for people trying to access Tidepool on Safari. Sorry about that one, that's been fixed.  
+[Feature Requirements](https://trello.com/c/GbuHSNpr/13-csp-error-in-safari)
+
+While we are working on an update to Tidepool Mobile, we noticed a bug that caused Tidepool Web to crash. It's nothing you would have encountered since you are running a test build of Tidepool Mobile, but we fixed this nonetheless.
```
Comment (gniezen) [inline _docs/release-notes.md]: should `since` be `unless`?

File: _docs/release-notes.md [hunk 4]
```diff
@@ -405,6 +420,15 @@ Visit [https://tidepool.org/uploader](https://tidepool.org/uploader) to download
 
 If you already have the Tidepool Uploader installed on your computer, it will automatically update to the latest version.  
 
+### 2.7.6 (Released 2018-09-13)
+We fixed a bug in our blob storage service that tried to send an empty data field to our database. Why send an empty box when you can just save on postage and send nothing at all?
```
Comment (gniezen) [inline _docs/release-notes.md]: I like this one! :laughing:

File: _docs/release-notes.md (hunk 1 above)
Comment (christopherasnider) [reply _docs/release-notes.md]: Good find.

File: _docs/release-notes.md (hunk 2 above)
Comment (christopherasnider) [reply _docs/release-notes.md]: Good thinking. I will be more specific.

File: _docs/release-notes.md (hunk 3 above)
Comment (christopherasnider) [reply _docs/release-notes.md]: I'm good with that suggestion. Updated.

**Pull request merged**